package.json

```diff
@@ -53,9 +53,9 @@
     "webpack-dev-server": "^3.2.1"
   },
   "peerDependencies": {
-    "@material-ui/core": "^3.0.0",
-    "react": "^16.4.1",
-    "react-dom": "^16.4.1"
+    "@material-ui/core": "^4.0.0",
+    "react": "^16.10.2",
+    "react-dom": "^16.10.2"
```

Comment:
Let's make the peer dependencies for react and react-dom to be `^16.8.0`, just enough to support react hooks.